File: Pemrograman-1/Modul 3/Python/PRAK305-2310817210014-DamarjatiSuryoLaksono.py
detik = int(input(""))

menit = detik // 60
jam = menit // 60
hari = jam // 24

if menit >= 60:
    menit = menit % 60

if detik >= 60:
    detik = detik % 60

if jam >= 24:
    jam = jam % 24
    print(f"{hari} hari", end=" ")

print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: Semester-1/Pemrograman-1/Modul-3/C/PRAK305-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int main() {
    int detik;
    scanf("%d", &detik);

    int menit = detik / 60;
    int jam = menit / 60;
    int hari = jam / 24;

    if (menit >= 60) {
        menit = menit % 60;
    }

    if (detik >= 60) {
        detik = detik % 60;
    }

    if (jam >= 24) {
        jam = jam % 24;
        printf("%d hari ", hari);
    }

    printf("%.2d:%.2d:%.2d", jam, menit, detik);

    return 0;
}
